samples/04_gis_analysts_data_scientists/chennai_floods_analysis.ipynb

```diff
@@ -790,7 +790,7 @@
    "source": [
```

Comment:
<p>Let's have a look at the major lakes and water reservoirs that were filled to the brim in Chennai due the rains. We plot the locations of some of the reservoirs that had a large outflow during the rains:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>